annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -91,10 +84,10 @@ protected ResponseEntity<String> storeAnnotation(MotivationTypes motivation, boo
 	    // 2. validate
 	    // check whether annotation with the given provider and identifier
 	    // already exist in the database
-	    if (annoId.getIdentifier() != null && getAnnotationService().existsInDb(annoId))
+	    if (getAnnotationService().existsInDb(annoIdentifier))
```

Comment:
this call should simply check that no id is provided in the body for the create method. it is not allowed anymore to submit annotation ids, as all of them are generated internally